ci/travis_before_script_cpp.sh

```diff
@@ -75,14 +75,19 @@ else
 fi
 
 # Do the style checks
-$TRAVIS_MAKE stylecheck
+$TRAVIS_MAKE stylefix
+if ! git diff-index --quiet HEAD --; then
+  # The syle fix required some changes
+  echho "Stylecheck failed, requires the following fix"
```

Comment:
Typo - should be echo
Should we mention that stylefix will automatically fix the issues flagged?

Reply:
yeah, travis failed because of this. still fixing.